src/workerd/jsg/setup.c++

```diff
@@ -461,6 +462,12 @@ bool IsolateBase::allowWasmCallback(v8::Local<v8::Context> context, v8::Local<v8
   return self->evalAllowed;
 }
 
+bool IsolateBase::jspiEnabledCallback(v8::Local<v8::Context> context) {
+  IsolateBase* self =
+      static_cast<IsolateBase*>(context->GetIsolate()->GetData(SET_DATA_ISOLATE_BASE));
+  return true;
```

Comment:
Should we be checking for something here?

Reply:
Yes should be `return self->jspiEnabled;`.